src/Tizen.Multimedia.MediaPlayer/Player/StreamInfo.cs

```diff
@@ -73,13 +73,15 @@ public int BitRate
             set;
         }
 
+
+
         /// <summary>
         /// Returns a string that represents the current object.
         /// </summary>
         /// <returns>A string that represents the current object.</returns>
-        /// <since_tizen> 3 </since_tizen>
-        public override string ToString() =>
-            $"SampleRate={ SampleRate.ToString() }, Channels={ Channels.ToString() }, BitRate={ BitRate.ToString() }";
+        /// <since_tizen> 3 </since_tizen>   
+	public override string ToString() =>
+            $"SampleRate={ SampleRate.ToString() },  Channels={ Channels.ToString() }, BitRate={ BitRate.ToString() }";
```

Comment:
The behavior of 'int.ToString()' could vary based on the current user's locale settings. Replace this call in 'AudioStreamProperties.ToString()' with a call to 'int.ToString(IFormatProvider)'.